index.js

```diff
@@ -669,7 +669,11 @@ class GithubScm extends Scm {
             });
         }
         if (type === 'repo') {
-            return hoek.reach(payload, 'head_commit.modified');
+            const added = hoek.reach(payload, 'head_commit.added');
+            const modified = hoek.reach(payload, 'head_commit.modified');
+            const removed = hoek.reach(payload, 'head_commit.removed');
+
+            return [...new Set([...added, ...modified, ...removed])];
```

Comment:
Non-blocking comment, but this may merit a comment explaining what's going on here.

Using `[...new Set([])]` as a way to prune duplicates is creative and certainly gets the job done, but it's not obvious at first glance.

Reply:
Added a comment 👍